skops/card/tests/test_card.py

```diff
@@ -403,6 +404,105 @@ def test_add_twice(self, model_card):
         assert text1 == text2
 
 
+def test_add_plot(destination_path, model_card):
```

Comment:
You probably added this test because of a merge with main. However, it can be deleted, since the functionality is already tested here:

https://github.com/skops-dev/skops/blob/1fd7982a3e2eb7071729189f74f3e23c9ebb4569/skops/card/tests/test_card.py#L857